content/blog/2019-08-08-commcall-sep2019.md

```diff
@@ -0,0 +1,47 @@
+---
+slug: commcall-sep2019
+title: Community Call - Reproducible Workflows at Scale with drake
+date: '2019-08-08'
+authors:
+  - Stefanie Butland
+categories: blog
+topicid:
+tags:
+  - community
+  - events
+  - community call
+  - reproducibility
+  - reproducible-research
+  - workflow
+  - pipeline
+  - high performance computing
+  - drake
+  - R
+---
+Ambitious workflows in R, such as machine learning analyses, can be difficult to manage. A single round of computation can take several hours to complete, and routine updates to the code and data tend to invalidate hard-earned results. You can enhance the maintainability, hygiene, speed, scale, and reproducibility of such projects with the drake R package. drake resolves the dependency structure of your analysis pipeline, skips tasks that are already up to date, executes the rest with optional distributed computing, and organizes the output so you rarely have to think about data files. This talk demonstrates how to create and maintain a realistic machine learning project using drake-powered automation.
+
+This 1-hour Community Call will include a presentation by drake developer, Will Landau, and at least 20 minutes for Q & A.
+
+🎤 See speaker bio below.  
+
+#### Join the Call
+
+🕘 **Tuesday, September 24, 2019, 9-10 AM PDT** ([find your local time](http://bit.ly/2TaBfCo))
+
+☎️ Everyone is welcome. No RSVP needed. Details to join the Call will be added here one week prior to the event.
+
+🎥 After the Call, we’ll post the video and collaborative notes on the [archive page](/commcalls/2019-09-24/).
+
+#### Resources
+
+- The [drake](https://github.com/ropensci/drake) R package
+- drake [User Manual](https://ropenscilabs.github.io/drake-manual/)
+- Learn drake [workshop](https://github.com/wlandau/learndrake)
+
+#### Speaker
+
+<img src="/img/blog-images/2019-08-08-commcall-sep2019/will-landau.jpg" alt="headshot of Will Landau" style="margin: 0px 20px; width: 200px;" align="left">
+_**Will Landau** is ...._  
+
+{{< figure src="/img/blog-images/2019-08-08-commcall-sep2019/will-landau.jpg" alt="headshot of Will Landau" width="200" align="left">}} _**Will Landau** does statistics and writes software in the life sciences industry. He is a Research Statistician at Eli Lilly and Company. Will has a PhD in Statistics._  
```

Comment:
How about this:

**Will Landau** is a Research Scientist at Eli Lilly and Company, where he develops  methods and software for statisticians. Prior to joining Lilly, he earned his PhD in Statistics from Iowa State University.